Typos on MonetaryPolicy.md

Derndingle opened the pull request.

Derndingle committed:
Typos on MonetaryPolicy.md

gititGoro commented:
Thank you for the typos. At paragraph 471, it's not clear what changed. I think your indentation changed. Could you revert to the original spacing and just change the words which are incorrect? 

At the beginning of 471, it says 
"So let's start on at the beginning..."

Should this not be
"So let's start at the beginning..."?

We could probably lose the So as well.

Derndingle committed:
Update MonetaryPolicy.md

I added the changes you suggested, The original change I made was from, "the smith can gradually phase out to holding of gold coins"
to, "the smith can gradually phase out the holding of gold coins".

Not sure why that change wasn't highlighted like the others

Derndingle commented:
I added the changes you suggested, The original change I made was from, "the smith can gradually phase out to holding of gold coins"
to, "the smith can gradually phase out the holding of gold coins".

Not sure why that change wasn't highlighted like the others
